src/test/fuzz/process_message.cpp

```diff
@@ -77,6 +77,10 @@ void fuzz_target(const std::vector<uint8_t>& buffer, const std::string& LIMIT_TO
         LOCK(p2p_node.cs_sendProcessing);
         g_setup->m_node.peerman->SendMessages(&p2p_node);
     }
+    {
```

Comment:
looks strange, 2 times same snippet here, but missing changes in process_messages.cpp.

Seems as conflict resolving bug